templates/migration-wrapper.sql

```diff
@@ -0,0 +1,25 @@
+-- {{filename}}
+DO
+$block1$
```

Comment:
Thinking maybe we should use more obscure string delimiters so we don't potentially clash with the body of the migration. Maybe `$outer_migration_block$` and `$inner_migration_block$`?

Reply:
👍